journal/vendor-extra/PatternsBundle/Resources/views/views.html.twig

```diff
@@ -0,0 +1,10 @@
+{% for view in views -%}
+
+    {% if view.template is defined %}
+        {%- include view.template with view.arguments only -%}
+    {% else %}
+        {#- Assuming plain string -#}
+        {{- view -}}
```

Comment:
Not sure about this. Added for https://github.com/libero/walking-skeleton/blob/c28487fb3d36115eb0d4996d5c4680edbab0b108/journal/src/Controller/HomeController.php#L57 so it didn't have to manually create `text` views.